data/core/encyclopedia/geography.cfg

```diff
@@ -117,69 +149,89 @@ Over the River Aethen, south of Fort Tahn, is a Wesnothian frontier region. It i
-
-<header>text='Notable cities:'</header>
-    • Barag Gor, a city home to the Orcish Council
-    • Bitok
-    • Borstep
-    • Farzi
-    • Lmarig
-    • Melmog
-    • Prestim
-    • Tirigaz
-    • Dorest, the northernmost human city
+    text= _ "Cold, harsh, and inaccessible, the Far North is the ancestral home of the Orcish Clannate. It lies north of the <ref>dst='heart_mountains' text='Heart Mountains'</ref>, which the Orcs call the Haggid-Dargor and claim (without merit) as their own. To the east lie the Unaligned Tribes of the Wild Steppe, who fell out of the control of the Clannate, instead roaming with wild human barbarians and clashing with the High Elves of the North Plains (known as North Elves in human lands). The High Elves themselves reside further east, where it is rumored they rule a vast kingdom." + "
+" +
+        #po: Geography of the Far North
+        _ "<header>text='Notable cities:'</header>
+               • Barag Gor: A city home to the Orcish Council.
+               • Bitok
+               • Borstep
+               • Farzi
+               • Jotha: A large fortified capital of the local merfolk that lies within the bay of Jotha.
+               • Lmarig
+               • Melmog
+               • Prestim
+               • Tirigaz: A coastal orcish town that occasionally raids Jotha.
+               • Dorest: The northernmost human city." + "
+" +
+        #po: Geography of the Far North
+        _ "<header>text='Notable land features:'</header>
+               • Desert of Death: An unsually dry desert in the Far North filled with venomous Sand Scuttlers. It is extremely hot by day yet chilly by night.
+               • Black Marshes: A perilous swamp north of Tirigaz filled with violent monsters.
+               • Mountains of Dorth: A set of mountains north of the Heart Mountains. Though nowhere near as dangeous as its southern neighbour, it is nonetheless deadly to inexperienced travelers.
+               • Mountains of Haag: A group of high mountains inhabited by groups of orcs, trolls and dwarves.
+               • Greenwood: An elvish forest located near the Lintanir Forest. Some say it was once connected to it, until groups of savage orcs burned much of the forest.
+               • Silent Forest: A dense forest inhabited by a few elves, once known as Gitamoth.
+               • Forest of Thelien: A dense coniferous forest sometimes inhabited by elves.
+               • River Oumph: A river flowing from the Mountains of Haag to its delta just south of Bitok.
+               • River Bork: A river of the Far North flowing from a lake next to the Mourned Hills.
```

Comment:
This is the sole mention of Mourned Hills as far as I can tell.

Reply:
I think it miht be, though I used it explain the location better. It is found in the SotBE map, somit is a thing. Coudl be removed with a further revision later on.

Reply:
Could be removed, sure. Or an extra line about it could be added.